models/crypto.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const userModel = firestore.collection('Users')
+
+/**
+ * Fetches the data about our members
+ * @return {Promise<userModel|Array>}
+ */
+
+/**
+ * Fetches the user data from the the provided username or userId
+ *
+ * @param { Object }: Object with username and userId, any of the two can be used
+ * @return {Promise<{userExists: boolean, user: <userModel>}|{userExists: boolean, user: <userModel>}>}
+ */
+const fetchUser = async (name) => {
+  try {
+    let userData, id
+    if (name) {
+      const user = await userModel.where('name', '==', name).get()
+
+      user.forEach(doc => {
```

Comment:
Why are we iterating over user array?
Isn't is supposed to be singular? Otherwise doesn't it mean db is inconsistent?

Reply:
will check on it.

Reply:
That is the standard way to do get queries as stated by firestore documentation